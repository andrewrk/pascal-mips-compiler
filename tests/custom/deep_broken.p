program Main;
class Main begin
    function Main;
        var a : boolean;
    begin
        b := true;
        if b then
            if b then
                if b then
                    b := false;
        b := false;
        while b do
            while b do
                while b do
                    b := false;
        begin
            begin
                begin
                    begin
                        b := true;
                    end
                end
            end
        end;

        if b then begin
            if b then begin
                b := false
            end else begin
                b := true
            end
        end else begin
            if b then begin
                b := false
            end else begin
                b := true
            end
        end;
    end
end
.
