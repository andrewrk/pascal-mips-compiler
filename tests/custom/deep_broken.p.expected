ERROR: line 6: variable "b" not declared
ERROR: line 7: variable "b" not declared
ERROR: line 8: variable "b" not declared
ERROR: line 9: variable "b" not declared
ERROR: line 10: variable "b" not declared
ERROR: line 11: variable "b" not declared
ERROR: line 12: variable "b" not declared
ERROR: line 13: variable "b" not declared
ERROR: line 14: variable "b" not declared
ERROR: line 15: variable "b" not declared
ERROR: line 20: variable "b" not declared
ERROR: line 26: variable "b" not declared
ERROR: line 27: variable "b" not declared
ERROR: line 28: variable "b" not declared
ERROR: line 30: variable "b" not declared
ERROR: line 33: variable "b" not declared
ERROR: line 34: variable "b" not declared
ERROR: line 36: variable "b" not declared

